sdk/typescript/package.json

```diff
@@ -42,6 +42,11 @@
 			"source": "./src/keypairs/secp256r1/index.ts",
 			"import": "./dist/esm/keypairs/secp256r1/index.js",
 			"require": "./dist/cjs/keypairs/secp256r1/index.js"
+		},
```

Comment:
Noting to myself that this is how you add a new package

Reply:
This isn’t a package it’s just a way to more granularly import out of the package. It’s an approach we’re trying to take to minimize the required size of the SDK.